File: Python/Constructor/Nomina.py
#Esta clase es la que va a llevar el constructor, con sus atributos y la invocacion de los mismos
class Nomina:

    def __init__(self, nombre, apellidos, cedula, salario):#self referencia a la instancia actual
        self.nombre = nombre
        self.apellidos = apellidos
        self.cedula = cedula#referencia al self
        self.salario = salario

    def dato(self):#metodo el cual imprimire los datos
        print(" ")
        print("El empleado siguiente es ")
        print(" ")
        print("* El nombre del empleado es: ", self.nombre)#concateno con las variables
        print("* El apellido del empleado es: ", self.apellidos)
        print("* La cedula del empleado es: ", self.cedula)
        print("* El salario del empleado es: ", self.salario)

File: Java/Constructor/Nomina.java
/*
 * Esta clase es la que va a llevar el constructor, con sus atributos 
 * y la invocacion de los mismos
 */
package constructor;
/**
 * 
 * @author Harold Pedraza
 * @version 15 de noviembre del 2017
 */

public class Nomina {
    //crear atributos
    String nombres, apellidos;
    int cedula, salario;
    
    //creando un metodo constructor
    
    Nomina(String nombre, String apellido, int cedula, int salario){
        //inicializando los atributos
        this.nombres = nombre ;
        this.apellidos = apellido;
        this.cedula = cedula ;
        this.salario = salario;
    }
    

    
    void Datos(){
        System.out.println(" ");
        System.out.println("El empleado siguiente es ");
        System.out.println(" ");
        System.out.println("* El nombre del empleado es: "+nombres);
        System.out.println("* El apellido del empleado es: "+apellidos);
        System.out.println("* La cedula del empleado es: "+cedula);
        System.out.println("* El salario del empleado es: "+salario);
    }  
}
